x-pack/plugins/cases/server/telemetry/queries/utils.ts

```diff
@@ -121,6 +191,48 @@ export const getCountsFromBuckets = (buckets: Buckets['buckets']) => ({
   monthly: buckets?.[0]?.doc_count ?? 0,
 });
 
+export const getCountsAndMaxAlertsData = async ({
+  savedObjectsClient,
+  savedObjectType,
+  filter,
+}: {
+  savedObjectsClient: TelemetrySavedObjectsClient;
+  savedObjectType: string;
+  filter?: KueryNode;
+}) => {
+  const res = await savedObjectsClient.find<
+    unknown,
+    {
+      counts: Buckets;
+      references: MaxBucketOnCaseAggregation['references'];
+      uniqueAlertCommentsCount: { value: number };
+    }
+  >({
+    page: 0,
+    perPage: 0,
+    filter,
+    type: savedObjectType,
+    namespaces: ['*'],
+    aggs: {
+      ...getAlertsCountsAggregationQuery(savedObjectType),
+      ...getAlertsMaxBucketOnCaseAggregationQuery(savedObjectType),
+      ...getUniqueAlertCommentsCountQuery(savedObjectType),
+    },
+  });
+
+  const countsBuckets = res.aggregations?.counts?.buckets ?? [];
+  const totalAlerts = res.aggregations?.uniqueAlertCommentsCount.value ?? 0;
+  const maxOnACase = res.aggregations?.references?.cases?.max?.value ?? 0;
+
+  return {
+    all: {
+      total: totalAlerts,
+      ...getCountsFromBuckets(countsBuckets),
```

Comment:
I think the `getCountsFromBuckets` will not work for this use case because we need to access the nested `topAlertsPerBucket` aggregation inside of each bucket. I haven't tested, maybe I am wrong.

Reply:
You're right, I will add another function that will get the topAlertsPerBucket value for each case: daily, weekly, monthly.

Reply:
done, thanks!